### ISSUE TITLE
Invalid Role Definition for WAI-ARIA role.

### ISSUE THREAD

**Issue by pmccorison** [tangbc/vue-virtual-scroll-list#242]
The W3C spec for elements in a group states that all items below an item with a group role should be "listitem" elements, not "item" elements. I am seeing WCAG violations using your library based on this.

Can we get the default state of these roles changed, and/or add support to specify those roles? 

https://www.w3.org/TR/wai-aria-1.0/roles#group

**Comment by tangbc** [comment]
Just use standard offical recommands.
